lib/src/main/java/io/ably/lib/transport/Hosts.java

```diff
@@ -6,51 +6,61 @@
  * Created by gokhanbarisaker on 2/1/16.
  */
 public class Hosts {
-    private static final List<String> FALLBACKS = Defaults.HOST_FALLBACKS;
-    private static final String PATTERN_REST_PROD = "^rest.ably.io$";
-    private static final String PATTERN_REALTIME_PROD = "^realtime.ably.io$";
+	private static final List<String> FALLBACKS = Defaults.HOST_FALLBACKS;
+	private static final String PATTERN_REST_PROD = "^rest.ably.io$";
```

Comment:
I realise this is old code, but if we're just checking for this pattern why don't we simply use a string and check `host.equals(PATTERN_REST_PROD)`